wdl-format/tests/format/seaseq-case/source.formatted.wdl

```diff
@@ -0,0 +1,939 @@
+
+    parameter_meta {
+        reference: {
+            description: "Reference FASTA file",
+            group: "reference_genome",
+            patterns: [
+                "*.fa",
+                "*.fasta",
+                "*.fa.gz",
+                "*.fasta.gz",
+            ],
+        }
+        blacklist: {
+            description: "Blacklist file in BED format",
+            group: "reference_genome",
+            help: "If defined, blacklist regions listed are excluded after reference alignment.",
+            patterns: [
+                "*.bed",
+                "*.bed.gz",
+            ],
+        }
+        gtf: {
+            description: "gene annotation file (.gtf)",
+            group: "reference_genome",
+            help: "Input gene annotation file from RefSeq or GENCODE (.gtf).",
+            patterns: [
+                "*.gtf",
+                "*.gtf.gz",
+                "*.gff",
+                "*.gff.gz",
+                "*.gff3",
+                "*.gff3.gz",
+            ],
+        }
+        bowtie_index: {
+            description: "bowtie v1 index files (*.ebwt)",
+            group: "reference_genome",
+            help: "If not defined, bowtie v1 index files are generated, will take a longer compute time.",
+            patterns: [
+                "*.ebwt",
+            ],
+        }
+        motif_databases: {
+            description: "One or more of the MEME suite motif databases (*.meme)",
+            group: "reference_genome",
+            help: "Input one or more motif databases available from the MEME suite (https://meme-suite.org/meme/db/motifs).",
+            patterns: [
+                "*.meme",
+            ],
+        }
+        sample_sraid: {
+            description: "One or more sample SRA (Sequence Read Archive) run identifiers",
+            group: "input_genomic_data",
+            help: "Input publicly available FASTQs (SRRs). Multiple SRRs are separated by commas (,).",
+            example: "SRR12345678",
+        }
+        sample_fastq: {
+            description: "One or more sample FASTQs",
+            group: "input_genomic_data",
+            help: "Upload zipped FASTQ files.",
+            patterns: [
+                "*.fq.gz",
+                "*.fastq.gz",
+            ],
+        }
+        results_name: {
+            description: "Experiment results custom name",
+            group: "analysis_parameter",
+            help: "Input preferred analysis results name (recommended if multiple FASTQs are provided).",
+            example: "AllMerge_mapped",
+        }
+        run_motifs: {
+            description: "Perform Motif Analysis",
+            group: "analysis_parameter",
+            help: "Setting this means Motif Discovery and Enrichment analysis will be performed.",
+            example: true,
+        }
+    }
+
+    input {
+        # group: reference_genome
+        File reference
+        File? spikein_reference
+        File? blacklist
+        File gtf
```

Comment:
So you're not currently sorting inputs. Looking at this SEAseq example, I do wonder if the formatter will annoy users if they're doing something like this and using comments and spacing to group things.

Reply:
Yes, sorting inputs is an outstanding TODO, but I also think you're right that it might annoy some people (many people?). I'm thinking we want to provide an easy "off-switch" for that specific behavior. A dedicated `--no-input-sorting` flag.

Reply:
I suppose since our formatter is opinionated, having it opt-out is fine. There may be features that should be opt-in instead though.

Reply:
Like what? I think input sorting may be our most controversial. If that's opt-out, I would think everything else is as well.

Reply:
Something like sorting arrays in `parameter_meta` (or generally). That's a fairly easy thing for the formatter to do, but should be opt-in.